functions-and-operators/string-functions.md

```diff
@@ -107,19 +107,87 @@ Return a string containing binary representation of a number.
 
 ### [`BIT_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_bit-length)
 
-Return length of argument in bits.
+The `BIT_LENGTH()` function is used to return the lenght of a given string in bits. It counts the total number of characters in a given string and then provides the result in bits.
+
+Examples:
+
+```sql
+SELECT BIT_LENGTH("TiDB");
++----------+
+| BIT_LENGTH("TiDB") |
++----------+
+| 8 (T) + 8 (i) + 8 (D) + 8 (B) = 40 bits |
++----------+
+```
+```sql
+SELECT CustomerName, BIT_LENGTH(CustomerName) AS BitLengthOfName FROM Customers;
+| CustomerName|BitLengthOfName |
+|-------------|----------------|
+| Albert Einstein | 120 bits |
+| Robert Oppenheimer | 144 bits |
+```
+
+> **Note:**
+>
+> The second example operates under the assumption that there is a database with a record titled `Customers` and a field inside titled `CustomerName`
 
 ### [`CHAR()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_char)
 
-Return the character for each integer passed.
+The `CHAR()` function is used to get the character value of a specific number. It performs the opposite function of `ASCII()`. Unlike `ASCII()`, which can get the ASCII value of a specific character, `CHAR()` can get the character of a specific ASCII value.
```

Comment:
A note about how it (isn't) handling non-ASCII characters would be good.

```
mysql> SELECT ASCII("🏔");
+---------------+
| ASCII("🏔")     |
+---------------+
|           240 |
+---------------+
1 row in set (0.00 sec)

mysql> SELECT CHAR(240);
+----------------------+
| CHAR(240)            |
+----------------------+
| 0xF0                 |
+----------------------+
1 row in set (0.00 sec)
```

Reply:
Maybe link to https://en.wikipedia.org/wiki/ASCII

Reply:
é is C3A9, which is 50089 in decimal

```
mysql> SELECT CHAR(50089);
+-------------+
| CHAR(50089) |
+-------------+
| é           |
+-------------+
1 row in set (0.00 sec)

mysql> SELECT HEX('é');
+-----------+
| HEX('é')  |
+-----------+
| C3A9      |
+-----------+
1 row in set (0.00 sec)

mysql> SELECT CHAR(0xC3A9);
+--------------+
| CHAR(0xC3A9) |
+--------------+
| é            |
+--------------+
1 row in set (0.01 sec)
```

Note that `mysql --skip-binary-as-hex ...` is needed for this to work unless you cast it to a string.